newIDE/app/src/Profile/Subscription/SubscriptionDialog.js

```diff
@@ -386,6 +422,47 @@ export default function SubscriptionDialog({
               />,
             ]}
             open={open}
+            fixedContent={
+              <>
+                <Line justifyContent="space-between" alignItems="center">
+                  <Text size="block-title">
+                    <Trans>Subscription plans</Trans>
+                  </Text>
+                  <TwoStatesButton
+                    value={period}
+                    leftButton={{
+                      label: <Trans>Monthly</Trans>,
+                      value: 'month',
+                    }}
+                    rightButton={{
+                      label: <Trans>Yearly</Trans>,
+                      value: 'year',
+                    }}
+                    // $FlowIgnore
```

Comment:
why? can't this be better typed?

Reply:
I could use a Generic but Flow does not support Generic with forwardRef unfortunately